beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -225,13 +310,64 @@ def normalize(tensor):
 plt.imshow(fbank.t().numpy(), cmap='gray')
 
 
+######################################################################
+# You can create mel frequency cepstral coefficients from a raw audio signal
+# This matches the input/output of Kaldi’s compute-mfcc-feats.
+# 
+
+mfcc = torchaudio.compliance.kaldi.mfcc(waveform, **params)
+
+print("Shape of mfcc: {}".format(mfcc.size()))
+
+plt.figure()
+plt.imshow(mfcc.t().numpy(), cmap='gray')
+
+
+######################################################################
+# Available Datasets
+# -----------------
+# 
+# If you do not want to create your own dataset to train your model, ``torchaudio`` offers a
+# unified dataset interface. This interface supports lazy-loading of files to memory, download 
+# and extract functions, and datasets to build models.
+# 
+# The datasets ``torchaudio`` currently supports are:
+#
+# -  **VCTK**: Speech data uttered by 109 native speakers of English with various accents
+#    (`Read more here <https://homepages.inf.ed.ac.uk/jyamagis/page3/page58/page58.html>`_).
+# -  **Yesno**: Sixty recordings of one individual saying yes or no in Hebrew; each
+#    recording is eight words long (`Read more here <https://www.openslr.org/1/>`_).
+# -  **Common Voice**: An open source, multi-language dataset of voices that anyone can use
+#    to train speech-enabled applications (`Read more here <https://voice.mozilla.org/en/datasets>`_).
+# -  **LibriSpeech**: Large-scale (1000 hours) corpus of read English speech (`Read more here <http://www.openslr.org/12>`_).
+# 
+
+yesno_data = torchaudio.datasets.YESNO('~/datasets/YESNO/', download=True)
+
+# The data in YESNO gives us the output tuple (image, target) where target is index of the target class.
```

Comment:
"A data point in Yesno is a output tuple (waveform, sample_rate, labels) where labels is a list of integers with 1 for yes and 0 for no."